utopia-remix/app/models/project.server.spec.ts

```diff
@@ -418,16 +418,16 @@ describe('project model', () => {
       expect(got.projects[0].proj_id).toBe('seven')
       expect(got.projects[1].proj_id).toBe('four')
 
-      expect(Object.keys(got.collaborators).length).toBe(2)
-      expect(Object.keys(got.collaborators)[0]).toBe('seven')
-      expect(got.collaborators['seven'].length).toBe(2)
-      expect(got.collaborators['seven'].map((c) => c.id)[0]).toBe('bob')
-      expect(got.collaborators['seven'].map((c) => c.id)[1]).toBe('carol') // the owner is included implicitly!
-      expect(Object.keys(got.collaborators)[1]).toBe('four')
-      expect(got.collaborators['four'].length).toBe(3)
-      expect(got.collaborators['four'].map((c) => c.id)[0]).toBe('bob')
-      expect(got.collaborators['four'].map((c) => c.id)[1]).toBe('carol')
-      expect(got.collaborators['four'].map((c) => c.id)[2]).toBe('alice') // the owner is included implicitly!
+      expect(Object.keys(got.collaborators.byProjectId).length).toBe(2)
+      expect(Object.keys(got.collaborators.byProjectId)[0]).toBe('seven')
```

Comment:
We shouldn't be testing the order of the keys here. Instead it would be better to test `got.collaborators.byProjectId.hasOwnProperty('seven')`

Reply:
ah makes sense, updating